connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -6,20 +6,18 @@ import java.net.URI
 import java.nio.charset.StandardCharsets
 import java.security.InvalidParameterException
 import java.util
-import java.util.concurrent.TimeUnit
 import java.util.zip.GZIPOutputStream
 import java.util.{TimeZone, UUID}
 
-import com.microsoft.azure.kusto.data.Client
 import com.microsoft.azure.kusto.ingest.IngestionProperties.DATA_FORMAT
-import com.microsoft.azure.kusto.ingest.result.{IngestionResult, IngestionStatus, OperationStatus}
+import com.microsoft.azure.kusto.ingest.result.IngestionResult
 import com.microsoft.azure.kusto.ingest.source.BlobSourceInfo
 import com.microsoft.azure.kusto.ingest.{IngestClient, IngestionProperties}
 import com.microsoft.azure.storage.StorageCredentialsSharedAccessSignature
 import com.microsoft.azure.storage.blob.{CloudBlobContainer, CloudBlockBlob}
 import com.microsoft.kusto.spark.authentication.KustoAuthentication
-import com.microsoft.kusto.spark.datasink
 import com.microsoft.kusto.spark.common.KustoCoordinates
+import com.microsoft.kusto.spark.datasink
```

Comment:
not sure what tool you are using , but might want to reorder imports ?

Reply:
intellij 
funny that this happened